django_cookies_samesite/user_agent_checker.py

```diff
@@ -26,6 +26,17 @@ def __init__(self, user_agent_string=""):
         self.user_agent_device = user_agent_parsed.get("device", dict())
         self.user_agent_string = user_agent_parsed.get("string", "")
 
+    @property
+    def user_agent_version_parts(self):
+        version_parts = []
+        for name in ["major", "minor", "patch"]:
+            try:
```

Comment:
Instead of 'try-catch', we have to use isnumeric() function.

Reply:
@browniebroke  Sorry, try-catch would be a good way of doing it as we have to convert the string to Unicode in python 2. So, I'll keep the try-catch 👌